src/PrestaShopBundle/Form/Admin/Configure/ShopParameters/Contact/ContactType.php

```diff
@@ -95,27 +99,31 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 ],
             ])
             ->add('email', EmailType::class, [
+                'label' => $this->trans('Email address', 'Admin.Global'),
+                'help' => $this->trans('Emails will be sent to this address.', 'Admin.Shopparameters.Help'),
                 'required' => false,
                 'constraints' => [
                     new Email([
                         'message' => $this->trans(
                             '%s is invalid.',
-                            [],
                             'Admin.Notifications.Error'
                         ),
                     ]),
                 ],
             ])
-            ->add('is_messages_saving_enabled', SwitchType::class)
+            ->add('is_messages_saving_enabled', SwitchType::class, [
+                'label' => $this->trans('Save messages?', 'Admin.Shopparameters.Feature'),
+                'help' => $this->trans('If enabled, all messages will be saved in the "Customer Service" page under the "Customer" menu.', 'Admin.Shopparameters.Help'),
+            ])
             ->add('description', TranslatableType::class, [
+                'label' => $this->trans('Description', 'Admin.Global'),
                 'type' => TextareaType::class,
                 'required' => false,
                 'options' => [
                     'constraints' => [
-                        new CleanHtml([
+                        new NoTags([
                             'message' => $this->trans(
-                                '%s is invalid.',
-                                [],
+                                'Field %s is invalid. Field must not contain HTML tags.',
```

Comment:
```suggestion
                                'The "%s" field is invalid. HTML tags are not allowed.',
```
If I am not mistaken, it will look like 'The description field is invalid. HTML tags are not allowed.'

Reply:
I wasn't sure if we needed the quotation marks there

Reply:
```The "description" field is invalid. HTML tags are not allowed.``` so yeah with quotations. Regarding if they are needed I am not sure. As far as if they are needed I am not sure, I see benefit of name being in quotations, but they also add extra noise.

Reply:
Mhm, I'd say we keep it. for now To stay consistent with other pages where we use this string, like in invoices form.

Reply:
It's difficult to answer that question because the behavior is different everywhere on the software right now. :( Either we keep it to stay consistent with this PR (https://github.com/PrestaShop/PrestaShop/pull/20243/files) or we modify it on both. I agree with you that they should be there (if we follow grammatical rules) but you're right to say that they are adding extra noise...

Reply:
@LouiseBonnard What would you do? 😉